includes/snmp.inc.php

```diff
@@ -651,6 +655,9 @@ function snmp_gen_auth(&$device)
     } elseif ($device['snmpver'] === 'v2c' or $device['snmpver'] === 'v1') {
         $cmd  = " -".$device['snmpver'];
         $cmd .= " -c '".$device['community']."'";
+        if ($vlan) {
+            $cmd .= '@' . $vlan;
```

Comment:
I tried adding '-n vlan-[vlanid]' to the command but I'm not getting the same output as if I did 'communitystring@[vlan]'

https://pastebin.com/Q7uxfx6W